src/main/java/org/jboss/xavier/sanitizer/XavierPayloadSanitizer.java

```diff
@@ -0,0 +1,169 @@
+package org.jboss.xavier.sanitizer;
+
+import com.jayway.jsonpath.*;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class XavierPayloadSanitizer
+{
+    private static String inputFileName;
+    private static String outputFileName;
+    private static String issuesConditionsJSONFileName;
+
+    public static void main(final String[] args)
+    {
+        List<String> argsList = Arrays.asList(args);
+        argsList.forEach(arg -> {
+            if( arg.equals("--input"))
+            {
+                inputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+            if( arg.equals("--output"))
+            {
+                outputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+
+            if( arg.equals("--issues"))
+            {
+                issuesConditionsJSONFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+        });
+
+        String jsonFileToBeSanitized = readInputFile(inputFileName, "Input file to sanitize");
```

Comment:
Just add a space to make final concatenated String readable
```suggestion
        String jsonFileToBeSanitized = readInputFile(inputFileName, "Input file to sanitize ");
```